core/component.go

```diff
@@ -118,6 +118,18 @@ func NewComponent(ctx log.Interface, serviceName string, announcedAddress string
 		go http.ListenAndServe(fmt.Sprintf(":%d", healthPort), nil)
 	}
 
+	if monitors := viper.GetStringMapString("monitor-servers"); len(monitors) != 0 {
+		component.Monitors = map[string]pb_noc.MonitorClient{}
```

Comment:
and I actually thought that the short form(`map[string]]pb_noc.MonitorClient{}`) is easier to read and it has no difference for the compiler, but I will do that as a convention then.